tests/discrepancy/test_cd.py

```diff
@@ -0,0 +1,201 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.ensemble import RandomForestClassifier
+
+from pywhy_stats import bregman, kcd
+
+seed = 12345
+rng = np.random.default_rng(seed)
+# number of samples to use in generating test dataset; the lower the faster
+n_samples = 150
+
+
+def single_env_scm(n_samples=200, offset=0.0):
```

Comment:
If we only use this withing this test, add an `_` to the method.

Reply:
Done